### PR TITLE
Abstract each package's functions and tests into separate files

### LINKED ISSUE #11: Abstract each package's functions and tests into separate files

### PR DESCRIPTION
**Pull request by lukehedger**
Closes #11

### CODE REVIEW

File: packages/identity-claims/test/createProfileMetaIdentityClaim.spec.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      claim: claimMessage,
+      issuer: issuer.id,
+      property: `profile.${subProperty}`,
+      signature:
+        '0x244a310ddb00eb3a29551075c6ca8c8624431b551f0f02c5cb52eea9261cd29a0f2da373715494716f666f5e5bcddbe5434bc0e5895a1ff1f21df1785bf34eee00',
+      subject: issuer.id,
+    }
+
+    expect(actual).toEqual(expected)
+  })
```
Comment (MrTibbles) [inline packages/identity-claims/test/createProfileMetaIdentityClaim.spec.js]: A failing test spec should be added. ie. "Should **not** return a valid ...."

File: packages/identity-claims/test/createVerifiableIdentityClaimObject.spec.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      claimHash:
+        '0xea39355900790422fb9eab06d2c6c25a2c1ce1bb6955276eeedf1055cc35623b',
+      claimMessage: claimMessage,
+      signature:
+        '0x11694fae74072cc5df26f8e2cfeb1ba90f1a0c2c03145904973237b2ee2e32324d80808e76b446f2ee76143c33e3588d868e893ddd33c079a452bb9d81155b8f00',
+      subject: subject,
+      accessToken: extraData.accessToken,
+    }
+
+    expect(actual).toEqual(expected)
```
Comment (MrTibbles) [inline packages/identity-claims/test/createVerifiableIdentityClaimObject.spec.js]: Same as above, a failing spec should be addedd

File: packages/identity-claims/test/createVerifiedIdentityClaimObject.spec.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    const expected = {
+      claim: claimMessage,
+      issuer: issuer.id,
+      property: property,
+      signature:
+        '0xf87b4075132ce1cbc61ea2059c1b134e70269c2b7445f2fd8b1a56ebe8c7fccd7ca219d8ca4eb0193ef36046175cf84318e413415e1c9f16f55aab28df40048301',
+      subject: subject,
+    }
+
+    expect(actual).toEqual(expected)
```
Comment (MrTibbles) [inline packages/identity-claims/test/createVerifiedIdentityClaimObject.spec.js]: And again, failing test spec to be added.

File: packages/identity-claims/test/verifyIdentityClaim.spec.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    const actual = metaIdentityClaims.verifyIdentityClaim(
+      address,
+      claimHash,
+      signature
+    )
+    const expected = true
+
+    expect(actual).toEqual(expected)
+  })
+})
```
Comment (MrTibbles) [inline packages/identity-claims/test/verifyIdentityClaim.spec.js]: All tests should check for expected failing behaviour.

**Comment by lukehedger** [comment]
Yes, 100% agree with this. Will raise issues for parameter validation (on all methods) and corresponding tests and then handle in separate PRs.

Please approve if happy with this 👍 .

**Comment by MrTibbles** [comment]
Validation FTW

**Pull request merged**